src/PerformanceTests/Tests/Categories/Base.cs

```diff
@@ -109,12 +109,7 @@ static void LaunchAndWait(Permutation permutation)
 
             using (var p = Process.Start(pi))
             {
-                var maxDurationInSeconds = 150;
-                if (!p.WaitForExit(maxDurationInSeconds * 1000))
-                {
-                    p.Kill();
-                    Assert.Fail($"Killed process because execution took more then {maxDurationInSeconds} seconds.");
-                }
+                p.WaitForExit();
```

Comment:
Should we add `p.CloseMainWindow();` so that it at least tries to end the process?
I'm not sure why killing it causes errors?